lib/releaselib.sh

```diff
@@ -910,6 +910,8 @@ release::docker::release () {
     "kube-proxy"
     "hyperkube"
   )
+  # ml_platforms should be in the os1/arch1,os2/arch2,os3/arch3 form
+  local -r ml_platforms= $(echo "${KUBE_SERVER_PLATFORMS[@]}" | sed "s/ /,/g")
```

Comment:
bash golf: I think you can do something like `platforms=("${KUBE_SERVER_PLATFORMS[@]/ /,}")` if you want to eliminate the `sed`.